docs/root/configuration/http_filters/jwt_authn_filter.rst

```diff
@@ -0,0 +1,214 @@
+.. _config_http_filters_jwt_authn:
+
+JWT Authentication
+==================
+
+This HTTP filter can be used to verify JSON Web Token (JWT). It will verify its signature, expiration time, audiences and issuer. If the JWT verification fails, its request will be rejected. If the JWT verification success, its payload could be forwarded to the upstream for further authorization if desired.
```

Comment:
There is a check for not before time too https://github.com/envoyproxy/envoy/blob/9bd3805bee6f36ddc747679ae94180b20c0d00af/source/extensions/filters/http/jwt_authn/authenticator.cc#L135